endpoints/openrtb2/auction.go

```diff
@@ -1338,23 +1342,37 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 		return nil, nil, errs
 	}
 
+	isAppRequest, err := checkIfAppRequest(requestJson)
+	if err != nil {
+		return nil, nil, []error{err}
+	}
```

Comment:
Looks good. I like that you've extracted the check to a method. To increase performance, I recommend moving this check within the `deps.cfg.GenerateRequestID ` check, such that we only bother to check for an app request if the feature is turned on. I usually don't recommend nesting, but in this case it's a performance improvement in certain cases.